### PR TITLE
ref(api): Consolidate CSRF value exposure to api.tsx

### PR DESCRIPTION
**Pull request by evanpurkhiser**
Also just do not want constants.tsx accessing window

### CODE REVIEW

**Review by scefali — APPROVED** [review]

**Review by priscilawebdev — APPROVED** [review]
lgtm

**Review by matejminar — APPROVED** [review]

File: src/sentry/static/sentry/app/api.tsx [hunk 1]
```diff
@@ -422,7 +426,7 @@ export class Client {
     const isSameOrigin = window.location.origin === absoluteUrl.origin;
 
     if (!csrfSafeMethod(method) && isSameOrigin) {
-      headers.set('X-CSRFToken', getCookie(CSRF_COOKIE_NAME) ?? '');
+      headers.set('X-CSRFToken', getCsrfToken() ?? '');
```
[the remark this answers is not in the document]
Comment (EvanPurkhiser) [reply src/sentry/static/sentry/app/api.tsx:429]: Wouldn't the only other falsy value be `0`?

[the remark this answers is not in the document]
Comment (billyvg) [reply src/sentry/static/sentry/app/api.tsx:429]: yeah and `false` (although `getCookie()` is probably unlikely to return those falsey values) -- maybe this is more applicable to L#144

[the remark this answers is not in the document]
Comment (EvanPurkhiser) [reply src/sentry/static/sentry/app/api.tsx:429]: Yeah thinking the same thing

**Pull request merged**